pom.xml

```diff
@@ -109,6 +109,12 @@
             <version>3.8.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-aop</artifactId>
+            <version>5.3.28</version>
```

Comment:
Should use org.springframework:spring-framework-bom:5.3.28 to align all spring framework component versions instead

Reply:
This is the only spring dependency in the project, and it's test-only, so the bom would seem overkill